docs/enterprise/about.md

```diff
@@ -6,29 +6,29 @@ description: What does the Pomerium Enterpise Console offer?
 
 # Pomerium Enterprise
 
-<!-- This paragraph introduces what Pomerium Enterprise is. -->
+Pomerium Enterprise is built on Pomerium Open Source. Pomerium Enterprise makes Pomerium easier to manage at scale, and adds additional functionality aimed at organizations with auditing, compliance, governance, and risk management needs.
 
 ## Features
 
 In addition to the capabilities provided by open-source Pomerium, Pomerium Enterprise provides:
 
 ### Management GUI
 
-The Pomerium Enterprise Console lets you view traffic and logs, define routes and policies, and organize your service access all from an intuitive web interface.
+Pomerium Enterprise lets you view traffic and logs, define routes and policies, and organize your service access from an intuitive web interface.
 
-![Overview animation of the Pomerium Enterprise Console](./img/console-overview.gif)
+![Overview animation of Pomerium Enterprise](./img/console-overview.gif)
 
 ### Programmatic API
 
-Integrate Pomerium into your workflows by managing configuration from the programming language or infrastructure management tool of your choice.
+Integrate Pomerium into your workflows by managing configuration from the programming language or infrastructure management tool of your choice.Everything that is manageable in the Management GUI is also able to driven programmatically an API.
```

Comment:
```suggestion
Integrate Pomerium into your workflows by managing configuration from the programming language or infrastructure management tool of your choice. Everything that is manageable in the Management GUI can also be driven programmatically through the API.
```

Reply:
This is my attempt to fix the grammar, but if it changes the content too much, maybe we should 1:1 on what's being said here.